src/app/import/components/upload/upload.component.ts

```diff
@@ -300,4 +303,13 @@ export class UploadComponent implements OnInit, OnChanges, OnDestroy {
     this.uploadSpecificHelper.selectedTokenizer = findName.name;
   }
 
+  prepareEmbeddings(): void {
+    let q, vc;
+    [q, vc] = this.projectApolloService.getEmbeddingSchema(this.projectId);
+    vc.pipe(first()).subscribe((embeddings: Embedding[]) => {
+      const hasNotGdprEmbeddings = embeddings.filter((e: Embedding) => e.platform == PlatformType.COHERE || e.platform == PlatformType.OPEN_AI).length > 0;
+      this.uploadSpecificHelper.hasNotGdprEmbeddings = hasNotGdprEmbeddings;
```

Comment:
ah it's just the name of the variable that might be misleading :D functionality is already like described above.

Maybe name "recalculationCosts" or something similar?